force-app/main/default/classes/SOQL.cls

```diff
@@ -39,9 +40,15 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Contact> GetContactsByFirstName (String name) {
+        Contact[] myContacts = [SELECT Name,Id from Contact WHERE FirstName = :name];
+        if (myContacts.size()>0) {
+            return myContacts;    
+        }
+        else {
+            return null;
+        }
```

Comment:
I would definitely return the results of the query or an empty list rather than null:
```suggestion
        return [SELECT Name,Id from Contact WHERE FirstName = :name];
```